dashboard/app/models/sections/email.rb

```diff
@@ -0,0 +1,29 @@
+# == Schema Information
+#
+# Table name: sections
+#
+#  id                :integer          not null, primary key
+#  user_id           :integer          not null
+#  name              :string(255)
+#  created_at        :datetime
+#  updated_at        :datetime
+#  code              :string(255)
+#  script_id         :integer
+#  course_id         :integer
+#  grade             :string(255)
+#  login_type        :string(255)      default("email"), not null
+#  deleted_at        :datetime
+#  stage_extras      :boolean          default(FALSE), not null
+#  section_type      :string(255)
+#  first_activity_at :datetime
+#  pairing_allowed   :boolean          default(TRUE), not null
+#
+# Indexes
+#
+#  fk_rails_20b1e5de46        (course_id)
+#  index_sections_on_code     (code) UNIQUE
+#  index_sections_on_user_id  (user_id)
+#
+
+class Email < Section
```

Comment:
I think I'd prefer this was called `EmailSection`, rather than just `Email` - but I assume we have to do this so we can map existing `login_type`s to these classes without a migration, correct?  I might prefer a switch-case in the sti methods in `section.rb`, just to avoid confusing constructions like `Email.find_by_name`, `Email.teacher`, `Email.add_student` down the road.